docs/getting-started/concepts/feature-repo.md

```diff
@@ -0,0 +1,13 @@
+# Feature Repository
```

Comment:
@achals oops just realized we already have a [page](https://docs.feast.dev/getting-started/architecture-and-components/feature-repository) for feature repos

we should make sure to merge this in with that page somehow

Reply:
I'll merge